src/domains/ecommerce-app/api/ecommerce-transactions-service/v1/_openapi.json.tpl

```diff
@@ -664,29 +660,6 @@
           "details"
         ]
       },
-      "PostePayAuthRequestDetails": {
```

Comment:
Great but the same work needs to be done on the openapi ecommerce for checkout. What do you think ?